### ISSUE TITLE
Multisig: Coldcard backup assumes all signers use the same derivation path

### ISSUE THREAD

**Issue by sohomang** [cryptoadvance/specter-desktop#870]
**TLDR**: Specter-Desktop creates a faulty Coldcard backup file because it assumes all signers use the same derivation path.

Using Specter-Desktop to create a 2-of-2 multisig setup between:
- Coldcard (`m/48h/0h/0h/1h`)
- Electrum hot wallet (`m/1h`)

Here's the .txt backup file I get in `Settings > Export > Save Coldcard file`:

```
# Coldcard Multisig setup file (created on Specter Desktop)
#
Name: TEST
Policy: 2 of 2
Derivation: m/48'/0'/0'/2'
Format: P2WSH
J84KF93F: xpub6ex[...]
C47BF394: xpub6uJ[...]
```

Coldcard won't import this backup because it only specifies one derivation path (`m/48h/0h/0h/1h`) and omits the other (`m/1h`). I edited the faulty backup file to this, which worked:

```
# Coldcard Multisig setup file (created on Specter Desktop)
#
Name: TEST
Policy: 2 of 2
Format: P2WSH

Derivation: m/48'/0'/0'/2'
J84KF93F: xpub6ex[...]

Derivation: m/1'
C47BF394: xpub6uJ[...]
```

Is there any way to support this automatically intead of having to manually edit the backup file?

**Comment by sohomang** [comment]
Seems like it's related to the Coldcard's [latest firmware update](https://blog.coinkite.com/version-3.2.1-released/):
> Now tracks derivation path for each co-signer and no longer assumes they all use a shared derivation path.

**Comment by stepansnigirev** [comment]
I started looking into new CC firmware and have the following problem:
if one of the cosigners has xpub but master fingerprint and derivations are unknown (i.e. Electrum), Coldcard fails to import such multisig wallet.
If I provide empty derivation it complains that depth is different from derivation, if I don't provide derivation it complains that derivation is unknown.
Looks like we need another hack for coldcard.

**Comment by k9ert** [comment]
I guess this is already fixed or has a workaround as descibed in #1052. Please reopen if necessary.
